### PR TITLE
Updated Readme, Developer-Config, and some strings

### PR DESCRIPTION
**Pull request by boldtrn**
This PR updates the Readme, Developer Config and some strings. Nothing major, but please let me know if I should add or change anything?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,10 +33,6 @@ This project stands under the Apache License.
 It was originally based on the [Mapbox Sample App](https://github.com/mapbox/mapbox-navigation-android/tree/master/app),
```
Comment (karussell) [inline README.md]: Here I would write: "It is a fork of the [Mapbox Sample App]..."

Comment (karussell) [reply README.md]: Or maybe we keep using the MIT to keep it simple?

**Review by karussell — APPROVED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (boldtrn) [reply README.md]: Sure, I don't mind using the MIT license. Maybe we do the license change in a different PR?

Comment (karussell) [reply README.md]: ok

**Pull request merged**